### PR TITLE
Check update doc

### PR DESCRIPTION
**Pull request by kellyshang**

### CODE REVIEW

File: docs/protocols/CTMeta-schemas/nft-schema.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ sidebar_position: 0
 title: Metadata schema for CoTA NFT definition
 type: object
 properties:
+  cotaid:
```
Comment (duanyytop) [inline docs/protocols/CTMeta-schemas/nft-schema.md]: `cota_id`

File: docs/protocols/cota_main.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The CoTA Cell simply consists of two necessary parts, the SMT root in the `cell
  # CoTA cell data structure
 data:
     version: byte
-    smt_root: byte32
+    smt_root: Option<byte32>
```
Comment (duanyytop) [inline docs/protocols/cota_main.md]: I think `byte32` is better and here is pseudo code, not real coding implementation

File: docs/protocols/cota_witness.md [hunk 3]
```diff
@@ -190,8 +205,8 @@ table TransactionProof {
 ```jsx
 # CoTA cell data structure
 data:
-    version: byte     // must be 0 or 1(V1)
-    smt_root: optional<byte32>
+    version: byte
+    smt_root: Option<byte32>
```
Comment (duanyytop) [inline docs/protocols/cota_witness.md]: I think byte32 is better and here is pseudo code, not real coding implementation

File: docs/protocols/cota_main.md (hunk 2 above)
Comment (kellyshang) [reply docs/protocols/cota_main.md]: updated

File: docs/protocols/cota_witness.md (hunk 3 above)
Comment (kellyshang) [reply docs/protocols/cota_witness.md]: updated

File: docs/protocols/CTMeta-schemas/nft-schema.md (hunk 1 above)
Comment (kellyshang) [reply docs/protocols/CTMeta-schemas/nft-schema.md]: updated

**Review by duanyytop — APPROVED** [review]

**Pull request merged**